### PR TITLE
Update arch diagram to match terminology

### PR DESCRIPTION
**Pull request by thibmeu**
Update Figure 1 to include components from the Terminology.

* Credentials is owned by an Issuer
* Collection of receipt is done by a Relying Party
* Make of Transparency Statement is done by an Auditor, which is a special Relying Party

The newly generated diagram looks as follow
architecture difference

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -374,18 +375,24 @@ The arrows indicate the flow of information.
...
-              |             v v           |
-              |    .--------+-+---------. |
-              |   / Verify Transparent /  |
-              |  /      Statement     /   |
-              | '--------------------'    |
-              v                           v
-     .--------+---------.      .----------+-----.
-    / Collect Receipts /      /   Replay Log   /
-   '------------------'      '----------------'
+              V                           |   |
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:378]:
```suggestion
              |                           |   |
              V                           |   |
```

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -347,9 +347,9 @@ The arrows indicate the flow of information.
  .----------.
 |  Artifact  |
  '-----+----'
-       v                 .-------------.
-  .----+----.           |  Credentials  |
- | Statement |           '----+--+-----'
+       v            +----------------------+
+  .----+----.       |  Issuer Credentials  |
+ | Statement |      +---------+--+---------+
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:352]:
```suggestion
       v             +--------------------+
  .----+----.        | Issuer Credentials |
 | Statement |       +--------+--+--------+
```

File: draft-ietf-scitt-architecture.md [hunk 3]
```diff
@@ -381,11 +381,15 @@ The arrows indicate the flow of information.
               |    .--------+-+---------. |
               |   / Verify Transparent /  |
               |  /      Statement     /   |
-              | '--------------------'    |
+              | '-----------+--------'|   |
+              |             | Auditor |   |
+              |             +---------+   |
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:386]:
```suggestion
              | '----------+---------+    |
              |            | Auditor |    |
              |            +---------+    |
```

File: draft-ietf-scitt-architecture.md [hunk 4]
```diff
@@ -381,11 +381,15 @@ The arrows indicate the flow of information.
...
+              |            +---------+    |
               v                           v
-     .--------+---------.      .----------+-----.
-    / Collect Receipts /      /   Replay Log   /
-   '------------------'      '----------------'
+  .-----------+----------.       .----------+-----.
+ /  Collecting Receipt  /       /   Replay Log   /
+'-------+--------------'|      '-+--------------'|
+        | Relying Party |        | Relying Party |
+        +---------------+        +---------------+
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:392]:
```suggestion
  .-----------+----------.    .-----------+------.
 /  Collecting Receipt  /    /    Replay Log    /
'------+---------------+    '--+---------------+
       | Relying Party |       | Relying Party |
       +---------------+       +---------------+
```

**Review by SteveLasker — APPROVED** [review]
Thanks, @thibmeu 
LGTM

Minor tweaks to fix the aasvg corners:
image

**Review by aj-stein — APPROVED** [review]

**Review by JAG-UK — APPROVED** [review]
Lgtm

**Comment by yogeshbdeshpande** [comment]
@thibmeu I have a slight concern with the diagram. Normally Replay log is explicitly done by Auditor so Under Replay Log, if the role is Auditor that is a more correct representation than a Generic Relying Party what you have currently!

**Comment by yogeshbdeshpande** [comment]
Also I have a concern, Verify Transparent Statements is a job of any Relying Party, not specifically for the Auditor?

Did you intend to attach, the Auditor role to Verify Transparent Statements?

**Review by yogeshbdeshpande — CHANGES_REQUESTED** [review]
Kindly answer these questions please before I can approve, happy to discuss face to face!

**Comment by thibmeu** [comment]
> Also I have a concern, Verify Transparent Statements is a job of any Relying Party, not specifically for the Auditor?

From the terminology
```
Auditor: checks the correctness and consistency of all Transparent Statements issued by a Transparency Service
Relying Party: consumes Transparent Statements, verifying their proofs and inspecting the Statement payload
```

I read this as the Auditor builds verified transparent statement, while the replay log is a specialised relying Party but not necessarily an auditor

Removing Auditor is also a possibility to not put too many entities, since it's relying parties anyway. What do you think?

**Comment by thibmeu** [comment]
@yogeshbdeshpande I've addressed your comments following a discusion we had in person
here is the updated diagram

Screenshot from 2024-11-04 11-41-47

**Review by yogeshbdeshpande — APPROVED** [review]
Thank you for making this important change!
Looks Good to Me

**Pull request merged**